packages/shorebird_cli/lib/src/shorebird_config_mixin.dart

```diff
@@ -9,7 +9,10 @@ import 'package:yaml/yaml.dart';
 import 'package:yaml_edit/yaml_edit.dart';
 
 mixin ShorebirdConfigMixin on ShorebirdCommand {
-  bool get hasShorebirdYaml => getShorebirdYaml() != null;
+  bool get hasShorebirdYaml {
+    final file = File(p.join(Directory.current.path, 'shorebird.yaml'));
+    return file.existsSync();
+  }
```

Comment:
This is more efficient, but I'm curious if there was a different motivation for rewriting this. 

Also, take or leave, but this could be a one-liner:

```dart
bool get hasShorebirdYaml => File(p.join(Directory.current.path, 'shorebird.yaml')).existsSync();
```

Reply:
Sure I'll update the motivation was to avoid parsing the shorebird.yaml file since this method was meant to just check whether the file exists.

Reply:
Just out of interest, why is that more efficient?

Reply:
I assume Bryan just meant we don't need to create a temporary variable.

Reply:
Yeah, sorry, by "this" I meant "checking that the file exists but not parsing it". The one-liner is functionally the same and simply a matter of preference.

Reply:
Ah, thanks 👍